spec/controllers/invitations_controller_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+
+describe InvitationsController, type: :controller do
```

Comment:
You don't have to specify `, type: :controller` if the feature spec is under `spec/controllers`